apmpackage/apm/changelog.yml

```diff
@@ -1,6 +1,11 @@
 # newer versions go on top
 #
 # change type can be one of: enhancement, bugfix, breaking-change
+- version: "8.1.0"
+  changes:
+    - description: REPLACE ME WITH ACTUAL CHANGE LOG
```

Comment:
I'm not entirely happy with this, but it seems that the `elastic-package` requires an entry here, otherwise:

```
Lint the package
Error: checking package failed: linting package failed: found 1 validation error:
   1. current manifest version doesn't have changelog entry

make: *** [check-package] Error 1
```

Reply:
Can you create an issue in the package-spec repo to discuss this lint check? Initially the packages were not supposed to be Elastic stack version aligned, so the hard requirement on a changenote made sense. It should be more lenient though for stack version aligned packages. Maybe something like an _empty_ changenote entry could be added?

We can keep the changelog entry for now.

Reply:
Opened https://github.com/elastic/package-spec/issues/254